scripts/cv_api_master.py

```diff
@@ -257,6 +257,49 @@
 #     ],
 # }
 
+LOSSES_MASTER = {
+    "path": "losses/",
+    "title": "Losses",
```

Comment:
KerasCV Losses

Reply:
This will anyways generate as `https://keras.io/api/keras_cv/losses/` and will be listed under KerasCV API like below.
Do you still want me to make changes as KerasCV Losses?


![image](https://github.com/keras-team/keras-io/assets/73069040/0c1052ee-a4cd-48f8-8996-cc4ac0ae14b2)